content/python/concepts/built-in-functions/terms/str/str.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: 'str()'
+Description: 'Takes in a value that can be converted into a string, and returns a copy of the value in the string datatype.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+Takes in a value that can be converted into a string, and returns a copy of the value in the `string` datatype. If nothing is passed into `str()`, then an empty string is returned.
+
+## Syntax
+
+```py
+str()
+```
+
+## Example 1
+
+The `str()` function can change the datatype of a `float` value, `314.0`, into a string:
+
+```python
+print(str(314.0))
+```
+
+## Example 2
+
+The `str()` function can also wrap around other type conversions. How could that be done in the example below?
```

Comment:
```suggestion
The `str()` function can also wrap around other type conversions.
```

Not sure if these examples are supposed to play out like problems for users to solve, or just demonstrations as we would expect from the documentation. I prefer the latter.